app/models/o_auth_info.rb

```diff
@@ -5,7 +5,7 @@ def initialize(response)
   end
 
   def username
-    info.fetch('name').delete(' ').downcase
+    info.fetch('name').delete(' ').gsub(/[^0-9a-z ]/i, '').downcase
```

Comment:
Is the empty space in this regex intentional? Spaces should already be removed by `delete(' ')`. We definitely don't want spaces in the username-- it will cause problems similar to the one this PR is trying to fix.